lib/mayaUsd/python/__init__.py

```diff
@@ -1,4 +1,8 @@
-from . import _mayaUsd
 from pxr import Tf
-Tf.PrepareModule(_mayaUsd, locals())
-del _mayaUsd, Tf
+if hasattr(Tf, 'PreparePythonModule'):
+    Tf.PreparePythonModule('_mayaUsd')
+else:
+    from . import _mayaUsd
+    Tf.PrepareModule(_mayaUsd, locals())
+    del _mayaUsd
+del Tf
```

Comment:
You'll see this same change in a few places. Basically I adapted the files to handle a change made in USD v21.08 where they introduced a new python function `PreparePythonModule`. So first check if that one exists and if so call it. Otherwise for previous versions of USD use the current PrepareModule.